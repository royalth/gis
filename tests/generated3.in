50 52
10 28
45 3
6 37
3 2
47 39
25 8
3 31
10 47
29 27
41 26
1 30
47 42
4 28
3 24
33 4
42 24
36 3
10 35
35 15
6 26
37 42
26 21
5 38
48 1
5 48
11 45
9 1
1 34
27 21
19 17
30 13
44 31
43 28
24 32
26 14
38 19
22 36
12 34
12 46
37 7
15 16
13 49
12 45
23 11
50 29
40 5
17 25
18 21
38 20
4 36
7 35
31 30
