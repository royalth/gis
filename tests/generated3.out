5
1 9 2
1 30 1
1 34 3
1 48 4
2 3 5
3 24 3
3 31 1
3 36 4
3 45 2
4 28 1
4 33 3
4 36 2
5 38 1
5 40 2
5 48 3
6 26 1
6 37 2
7 35 2
7 37 3
8 25 1
10 28 2
10 35 3
10 47 1
11 23 2
11 45 3
12 34 2
12 45 1
12 46 3
13 30 2
13 49 1
14 26 2
15 16 2
15 35 1
17 19 1
17 25 2
18 21 1
19 38 2
20 38 3
21 26 3
21 27 2
22 36 3
24 32 1
24 42 2
26 41 4
27 29 1
28 43 3
29 50 2
30 31 3
31 44 2
37 42 1
39 47 2
42 47 3
